### DISCUSSION TITLE
Alternativa à extensão CSS Scan

### DISCUSSION THREAD

**Discussion by Roger-Melo** [frontendbr/forum#1357] [Dicas]
Olá, pessoas! Conhecem alguma alternativa grátis à (maravilhosa) extensão [CSS Scan](https://getcssscan.com/)?
Ela permite copiar, em um clique, o CSS do elemento em questão:

![css](https://user-images.githubusercontent.com/29297788/51788492-ad096b80-2165-11e9-9ee1-e4f0f0f21ba1.gif)

**Comment by victorferreira** [comment]
Não conhecia essa extensão, valeu pro compartilhar . 😄

**Comment by felipefialho** [comment]
Boa dica :)

**Comment by laforetmoreno** [comment]
A um tempo atrás eu andei dando uma procurada, mas nao achei nada. Se alguem souber deixa a dica ai!

**Comment by didiraja** [comment]
Quando quero fazer isso, eu uso o "styles computed" do chrome mesmo

**Comment by napalmdev** [comment]
tem essa CSS Peeper, ela não faz a cópia de tudo como a CSS Scan, mas ela mostra os estilos por elemento e tem outras coisas como uma aba com todas as cores utilizadas no site e assets tbm, é muito boa

https://chrome.google.com/webstore/detail/css-peeper/mbnbehikldjhnfehhnaidhjhoofhpehk

**Comment by guivr** [comment]
Opa, galera! Fico muito feliz por vocês gostarem do meu trampo 😄 

Não posso dar a extensão de graça pra todos aqui se não não vou poder continuar viajando pelo mundo haha

Mas tenho um presente pra todos vocês, sim!
30% de desconto pra geral do Frontendbr 🎉  (Aproveitem que vou subir pra $20 em breve)

Só comprar por esse link:
https://gum.co/cssscan/frontendbr

PS: Eu também gravei todas as 50 horas de desenvolvimento dessa extensão, se quiserem assistir:
https://www.youtube.com/watch?v=OtsNNXpXcYs


Grande abraço a todos!

**Comment by napalmdev** [comment]
> Opa, galera! Fico muito feliz por vocês gostarem do meu trampo 
> 
> Não posso dar a extensão de graça pra todos aqui se não não vou poder continuar viajando pelo mundo haha
> 
> Mas tenho um presente pra todos vocês, sim!
> 30% de desconto pra geral do Frontendbr  (Aproveitem que vou subir pra $20 em breve)
> 
> Só comprar por esse link:
> https://gum.co/cssscan/frontendbr
> 
> PS: Eu também gravei todas as 50 horas de desenvolvimento dessa extensão, se quiserem assistir:
> https://www.youtube.com/watch?v=OtsNNXpXcYs
> 
> Grande abraço a todos!

Que massa @guivr show demais mano, vlwzão \o/
